src/DependencyInjection/Compiler/ExtensionCompilerPass.php

```diff
@@ -245,10 +314,12 @@ private function hasTrait(\ReflectionClass $class, string $traitName): bool
 
     /**
      * @phpstan-param class-string $class
+     * @phpstan-param class-string $adminClass
      */
-    private function isSubtypeOf(string $type, string $subject, string $class): bool
+    private function isSubtypeOf(string $type, string $subject, string $class, string $adminClass): bool
```

Comment:
I think `isSubtypeOf` is not the best name for this method, or? Especially since now we are even adding 1 more class string as argument. And it also checks for traits which is not really a sub-type?

As far as I see this method is used to check if we should add an extension to an admin or not. So maybe we can name it `isMatchingExtension` or similar? :man_shrugging: